streamwebs_frontend/streamwebs/models.py

```diff
@@ -680,3 +680,69 @@ def __str__(self):
     class Meta:
         verbose_name = 'canopy cover survey'
         verbose_name_plural = 'canopy cover surveys'
+
+
+@python_2_unicode_compatible
+class Soil_Survey(models.Model):
+    school = models.CharField(max_length=250)
+    teacher = models.CharField(max_length=250)
+    date = models.DateTimeField(default=timezone.now)
+    weather = models.CharField(max_length=250)
+    site = models.ForeignKey(Site, null=True, on_delete=models.CASCADE)
+
+    landscape_pos_choices = [
+        ('summit', 'Summit'),
+        ('slope', 'Slope'),
+        ('depression', 'Depression'),
+        ('large_flat', 'Large Flat Area'),
+        ('stream_bank', 'Stream Bank')
+    ]
+
+    cover_type_choices = [
+        ('bare_soil', 'Bare Soil'),
+        ('rocks', 'Rocks'),
+        ('grass', 'Grass'),
+        ('shrubs', 'Shrubs'),
+        ('trees', 'Trees')
+    ]
+
+    land_use_choices = [
+        ('urban', 'Urban'),
+        ('agricultural', 'Agricultural'),
+        ('recreation', 'Recreation'),
+        ('wilderness', 'Wilderness'),
+        ('other', 'Other')
+    ]
+
+    landscape_pos = models.CharField(max_length=1, default=None,
```

Comment:
Doesn't max_length refer to the length of the first element of each tuple? In [this example](https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.Field.choices) the max length is 2 because that's the length of each of the constants. Maybe you should register Soil Survey in admin.py or make object creation tests to see if anything breaks while it's like this.